.storybook/preview.js

```diff
@@ -8,6 +8,7 @@ import { MockedProvider } from '@apollo/client/testing';
 import '../src/design-system/components/Badge/Badge.css';
 import '../src/design-system/components/Button/Button.css';
 import '../src/design-system/components/InputField/InputField.css';
+import '../src/design-system/components/RadioInput/RadioInput';
```

Comment:
probably doesn't make a difference, but the above have `.css`

Reply:
<img width="385" alt="Screenshot 2023-10-13 at 13 40 49" src="https://github.com/public-convenience-ltd/toiletmap/assets/1771189/71975043-b379-4d7b-84af-5de37c8bef3d">

acutally, it might be important because the styles aren't showing in the Storybook ^
https://62f528e69250fdfc51153bb7-vtwxgyczro.chromatic.com/?path=/story/pages-loos-add--add